src/driverstation/include/ssh_ui.hpp

```diff
@@ -0,0 +1,771 @@
+												}
+											}
+										}
+										break;
+									case 2:  // 24-bit Color Mode
+										{
+											uint8_t red_level = parameters[i += 2].value;
+											uint8_t green_level = parameters[i += 2].value;
+											uint8_t blue_level = parameters[i += 2].value;
+											this->current_foreground_color = Color{red_level, green_level, blue_level, 255};
+										}
+										break;
+								}
+								break;
+							case 49: this->current_background_color = this->default_background_color; break;  // Default Background
+							case 90: this->current_foreground_color = Color{127, 127, 127, 255}; this->current_text_effects |= BRIGHT; break;  // Bright Black Foreground
+							case 91: this->current_foreground_color = Color{255,   0,   0, 255}; this->current_text_effects |= BRIGHT; break;  // Bright Red Foreground
+							case 92: this->current_foreground_color = Color{  0, 255,   0, 255}; this->current_text_effects |= BRIGHT; break;  // Bright Green Foreground
+							case 93: this->current_foreground_color = Color{255, 255,   0, 255}; this->current_text_effects |= BRIGHT; break;  // Bright Yellow Foreground
+							case 94: this->current_foreground_color = Color{ 92,  92, 255, 255}; this->current_text_effects |= BRIGHT; break;  // Bright Blue Foreground
+							case 95: this->current_foreground_color = Color{255,   0, 255, 255}; this->current_text_effects |= BRIGHT; break;  // Bright Magenta Foreground
+							case 96: this->current_foreground_color = Color{  0, 255, 255, 255}; this->current_text_effects |= BRIGHT; break;  // Bright Cyan Foreground
+							case 97: this->current_foreground_color = Color{255, 255, 255, 255}; this->current_text_effects |= BRIGHT; break;  // Bright White Foreground
+							case 100: this->current_background_color = Color{127, 127, 127, 255}; break;  // Bright Black Background
+							case 101: this->current_background_color = Color{255,   0,   0, 255}; break;  // Bright Red Background
+							case 102: this->current_background_color = Color{  0, 255,   0, 255}; break;  // Bright Green Background
+							case 103: this->current_background_color = Color{255, 255,   0, 255}; break;  // Bright Yellow Background
+							case 104: this->current_background_color = Color{ 92,  92, 255, 255}; break;  // Bright Blue Background
+							case 105: this->current_background_color = Color{255,   0, 255, 255}; break;  // Bright Magenta Background
+							case 106: this->current_background_color = Color{  0, 255, 255, 255}; break;  // Bright Cyan Background
+							case 107: this->current_background_color = Color{255, 255, 255, 255}; break;  // Bright White Background
+							default:
+								std::cout << "Unhandled SGR parameter: " << parameters[i].value << std::endl;
+								break;
+						}
+					}
+					break;
+				default:
+					// TODO: Temporary
+					std::cout << "Unknown escape code: \\x1B[";
+					for(int i = 0; i < this->current_csi_parameters.size(); i++){
```

Comment:
cast i to size_t to prevent compiler warning about signed/unsigned comparison

Reply:
See [40be54a](https://github.com/Colorado-School-of-Mines-Robotics-Club/MATE_ROV_Programming/commit/40be54aa21dde2c1cf1b5c6848df74e4cb01a223#diff-e4d02ec2f011f01f0159acd6f4bd5bf98eafde3389acf8a6ceb66a36aeac33a0L710-R710)